src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -87,6 +90,16 @@ public Routes(
     }
 
     /**
+     * Creates a {@link Flow} based on executing the routes asynchronously.
+     *
+     * @return a new {@link Flow}
+     */
+    public Flow<HttpRequest, HttpResponse, NotUsed> flow() {
+        return Flow.<HttpRequest>create()
```

Comment:
I'd like to move it to acting more like a flow than a function.  This is a first step in doing that.  It allows you to treat the class like a flow or an async handler function.